manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -227,6 +231,14 @@ spec:
               description: 'Secret containing the tls cert and key for the ingress,
                 content generated if not provided (default: tls-secret)'
               type: string
+            uiWorkerImage:
+              description: Image string used for the UI worker deployments By default
+                this is determined by the orchestrator pod
+              type: string
+            webserverWorkerImage:
```

Comment:
What do you think of changing this to `apiWorkerImage`?  In the core repo it is currently `webservice`.  It is API not httpd, correct?  I find the name confusing because the UI and httpd are also webservers and I suspect others will too.

Reply:
This is used for both API and remote console workers. I just named them after the actual image names as it's image specific, not necessarily worker specific.